mobile/examples/object_detections/iOS/README.md

```diff
@@ -0,0 +1,45 @@
+# ONNX Runtime Mobile object detection iOS sample application
```

Comment:
how about renaming the directory `mobile/examples/object_detections` to `mobile/examples/object_detection`?